### PR TITLE
[Term Entry] Java Strings: .trim() Method #1597

### PR DESCRIPTION
**Pull request by MiiPoikaj**
### Description

A new term entry of .trim() for the Java: Strings concept. 

This contribution is part of the completion of "Learn Git&GitHub" course in Codecademy.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/strings/terms/trim/trim.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+Title: '.trim()'
+Description: 'Removes leading and trailing whitespace from a string.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'String'
+  - 'Methods'
+  - 'Whitespace'
```
Comment (caupolicandiaz) [inline content/java/concepts/strings/terms/trim/trim.md:9]:
```suggestion
  - 'Characters'
```

File: content/java/concepts/strings/terms/trim/trim.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+```pseudo
+string.trim()
+```
+
+No parameters are required for the `.trim()` method.
+
+## Example
+
+The `.trim)` method is showcased in the following example:
```
Comment (caupolicandiaz) [inline content/java/concepts/strings/terms/trim/trim.md:27]:
```suggestion
The `.trim()` method is showcased in the following example:
```

File: content/java/concepts/strings/terms/trim/trim.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+```java
```
Comment (caupolicandiaz) [inline content/java/concepts/strings/terms/trim/trim.md:29]:
```suggestion
```java
// Example.java
```

File: content/java/concepts/strings/terms/trim/trim.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+class Example {
+  public static void main(String[] args) {
+    String greetings = "   Hello, Stranger!  ";
+    System.out.println(greetings.trim());
+  }
+}
+```
+
+In this example, the .trim() method removes leading and trailing whitespaces from the string greetings.
```
Comment (caupolicandiaz) [inline content/java/concepts/strings/terms/trim/trim.md:40]:
```suggestion
In the example below, the `.trim()` method removes leading and trailing whitespaces from the string `greetings`.
```

Comment (caupolicandiaz) [reply content/java/concepts/strings/terms/trim/trim.md:40]: I'd recommend replacing the sentence introducing the example with this one.

File: content/java/concepts/strings/terms/trim/trim.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+The output would be:
```
Comment (caupolicandiaz) [inline content/java/concepts/strings/terms/trim/trim.md:42]:
```suggestion
This will produce the following output:
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@MiiPoikaj Finished my initial review, just a handful of small edits.

File: content/java/concepts/strings/terms/trim/trim.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+class Example {
+  public static void main(String[] args) {
+    String greetings = "   Hello, Stranger!  ";
+    System.out.println(greetings.trim());
+  }
+}
+```
+
+In the example below, the `.trim()` method removes leading and trailing whitespaces from the string `greetings`.
```
Comment (caupolicandiaz) [inline content/java/concepts/strings/terms/trim/trim.md:41]: @MiiPoikaj I think my previous note may have been confusing. I was suggesting that you replace line 27 with 41. Having both is unnecessary, and 41 is the more descriptive of the two. Let me know if you have any additional questions.

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]

**Review by caupolicandiaz — APPROVED** [review]
@MiiPoikaj Looks good! This is ready for another review

**Review by HishamT — APPROVED** [review]
LGTM!

**Comment by yangc95** [comment]
🥳 @MiiPoikaj Thanks for this entry! Here's the live link: https://www.codecademy.com/resources/docs/java/strings/trim

Although, you are currently anonymous. If you'd like to make yourself public, ensure that your GitHub username is added to your Codecademy profile and set your visibility to "everyone."
Screen Shot 2023-02-14 at 7 36 51 PM

cc: @HishamT @caupolicandiaz

**Pull request merged**